dvc/repo/__init__.py

```diff
@@ -93,7 +93,9 @@ def find_dvc_dir(root=None):
         return os.path.join(root_dir, Repo.DVC_DIR)
 
     def remind_to_git_add(self):
```

Comment:
Also, @pared introduced `remind_to_track` method in https://github.com/iterative/dvc/pull/1673/files , so if scm is not git it simply won't do anything, so this change seems to be not needed anymore.